### PR TITLE
[release-notes] Notes for my changes in 5.0.

### PR DESCRIPTION
**Pull request by evincarofautumn**
Not really sure what counts as “in-depth” versus “bugfixes”.

### CODE REVIEW

File: docs/about-mono/releases/5.0.0.md [hunk 1]
```diff
@@ -25,6 +25,20 @@ Mono.Simd assembly is deprecated in this release, and it will be
 removed in +2 releases. Users should transition to using the Vector
 classes, which are part of the official .net class libraries.
 
+### Internal Documentation ###
+
+We have enabled Doxygen for documentation of runtime internals. Going
+forward, this should help us onboard new contributors and make the
+codebase easier to navigate and understand.
```
Comment (akoeplinger) [inline docs/about-mono/releases/5.0.0.md]: A link to http://www.mono-project.com/api/ would be nice here

Comment (evincarofautumn) [reply docs/about-mono/releases/5.0.0.md]: I couldn’t find the URL, thanks. Updated.

**Comment by lewurm** [comment]
previously, bugfixes was just an enumeration of bugs we fixed (duh), for example: https://github.com/mono/website/blob/gh-pages/docs/about-mono/releases/4.8.0.md#bug-fixes

That's why I moved @alexanderkyte's changes in my PR https://github.com/mono/website/pull/273

**Comment by akoeplinger** [comment]
Correct, we have a script that greps all links to bugzilla bugs in commits and generates that list.

**Comment by evincarofautumn** [comment]
Alright, moved everything to the same section.

**Pull request merged**